modules/api/current.js

```diff
@@ -70,12 +77,19 @@ module.exports = {
 
     // prepare steps
     const stepsMap = {};
-    testSteps.forEach((step) => {
-      const { pickleStepId, id } = step;
-      // skip hookId
+    testSteps.forEach((step, index) => {
+      const { pickleStepId, id, hookId } = step;
+
       if (pickleStepId) {
-        const { steps } = this.storage.getPickle(pickleId);
-        stepsMap[id] = steps.find((item) => item.id === pickleStepId);
+        const { steps: stepsData } = this.storage.getPickle(pickleId);
+        const stepData = stepsData.find((item) => item.id === pickleStepId);
+        stepsMap[id] = { ...stepData, type: TEST_ITEM_TYPES.STEP };
+      } else if (hookId) {
+        const { name } = this.storage.getHook(hookId);
+        stepsMap[id] = {
+          text: name || (index === 0 ? 'Before' : 'After'),
+          type: index === 0 ? TEST_ITEM_TYPES.BEFORE_TEST : TEST_ITEM_TYPES.AFTER_TEST,
```

Comment:
Can we calculate it once and store in meaningful variable?
F.e. `const isBeforeHook = index === 0;`.